migration/sql-files/062-workitem-id-unique-per-space.sql

```diff
@@ -0,0 +1,91 @@
+-- first, we ADD a new COLUMN for the 'natural id' with the same values as the 'id'
+ALTER TABLE work_items ADD COLUMN "number" integer;
+UPDATE work_items SET number = id;
```

Comment:
yes, the current ID is unique across the system so it will be unique per space, too. With the change here, however, the issue will not be re-numbered from `1` within each space.